app/gift-exchanges/[id]/page.tsx

```diff
@@ -50,6 +51,39 @@ export default function GiftExchangePage() {
     : false;
   const inviteLink = window.location.href; // Invite Link for Invite Card
 
+  useEffect(() => {
+    if (!session?.user.id || !id) return;
+
+    const supabase = createClient();
+
+    const channel = supabase
+      .channel('gift_suggestions_live')
+      .on(
+        'postgres_changes',
+        {
+          event: 'INSERT',
+          schema: 'public',
+          table: 'gift_suggestions',
+          filter: `gift_exchange_id=eq.${id}`,
+        },
+        (payload: any) => {
+          if (payload.new.giver_id !== session.user.id) return;
```

Comment:
Shorthand is nice, but it comes at the cost of legibility. 
```suggestion
          if (payload.new.giver_id !== session.user.id) {
            return;
         }
```